Automattic/woocommerce-payments
# feat: UPE payment methods onboarding fixes

Fixes #

#### Changes proposed in this Pull Request









- Adds the notice for EU payment methods
- Fixes the country name

Before:
![2021-07-16 15 53 01](https://user-images.githubusercontent.com/273592/126007355-2f73f431-8cf1-4dd2-9316-46939d2a571c.gif)


After:
![2021-07-16 15 52 17](https://user-images.githubusercontent.com/273592/126007294-2067c50d-58d1-4d64-989e-b7db0db25608.gif)


#### Testing instructions





- Ensure you have `_wcpay_feature_account_overview_task_list` with `'1'` in your `wp_options`
- Ensure "Enable UPE settings changes" is disabled
- Ensure "Enable UPE checkout" is enabled
- Ensure you have "Enable Customer multi-currency" enabled
- Ensure your store's currency is USD
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-admin&task=woocommerce-payments--additional-payment-methods
- The country code is fixed
- The currency notice is displayed when an EU payment method is selected

-------------------
